pkgs/racket-test/tests/racket/pregexp.rkt

```diff
@@ -0,0 +1,18 @@
+#lang racket/base
+
+(require rackunit)
+
+(check-equal?
+ (regexp-quote "ab-cd")
```

Comment:
Shouldn't this be `pregexp-quote`? Is this test file running in the CI?

Reply:
Yes, I overlooked that, and I'll push a repair. Probably only tests are `racket-test-core` run in GitHub CI.

Reply:
https://github.com/racket/racket/blob/master/.github/workflows/ci-pr.yml#L23 is what runs in CI. Probably more files should be added there.